### DISCUSSION TITLE
onnxUI.py: error: unrecognized arguments: --cpu-textenc

### DISCUSSION THREAD

**Discussion by M4CC** [azuritecoin/OnnxDiffusersUI#126] [General]
Hello,

I'm trying to run onnx with the --cpu-textenc option but I'm getting the following error:
onnxUI.py: error: unrecognized arguments: --cpu-textenc

Can anyone help me figure out why?

Thank you!

**Comment by uchuusen** [comment]
Can you share the full commandline you used to start onnxUI.py, and maybe the full error message too? It works fine for me with the latest one.

**Comment by M4CC** [comment]
This was what I ran:
(virtualenv) c:\onnx>python onnxUI.py --cpu-textenc
usage: onnxUI.py [-h] [--cpu-only] [--release-memory]
onnxUI.py: error: unrecognized arguments: --cpu-textenc
